### DISCUSSION TITLE
[Question]: Thoughts on enable/disable reviews

### DISCUSSION THREAD

**Discussion by blacknell** [blakeblackshear/frigate#13272] [Ask A Question] [answered]
### What is your question:

Any thoughts for how I might (via MQTT) enable and disable flagging detections for review.

My back garden camera is set up to detect and flag review the right type of activity but it generates far too much when we're just using it as a family. I'd like to keep recording events as now but only flag for review those events that happen when the house alarm is armed (via an MQTT publish command) - (similar to how I optimise contrast at sunrise/set)

**Comment by NickM-27** [comment]
There's a feature request for it already, seems to make sense in general

**Comment by blacknell** [comment]
Is this making its way into 0.15? I can't see anything in the preview docs

**Reply by NickM-27** [reply]
No it has not been implemented

**Comment by brian-mckinney** [comment]
Any update on this? It would be useful for automations.  ie. disable reviews on inside cameras when people are home

**Reply by NickM-27** [reply, marked as answer]
This was implemented in 0.16
